apps/dashboard/package.json

```diff
@@ -104,6 +104,7 @@
     "thirdweb": "workspace:*",
     "tiny-invariant": "^1.3.3",
     "use-debounce": "^10.0.1",
+    "viem": "^2.16.0",
```

Comment:
do we explicitly need to add it here? why?

Reply:
Yep this was a mistake, only need to upgrade it where it's already installed